asclepias_broker/harvester/crossref.py

```diff
@@ -92,6 +92,35 @@ def _clean_scholix(self, data):
             data[k]['Identifier']['IDURL'] = data[k]['Identifier'].pop('IDUrl')
         return data
 
+    def _parse_doi_from_url(self, url):
+        """."""
+        splitted_url = url.split('/')
+        doi = splitted_url[-2] + '/' + splitted_url[-1]
+        return doi
+
+    def _transform_events(self, data):
```

Comment:
This has to be called depending on if the events are Scholix or not

Reply:
Yup I haven't connected it yet since I was expecting feedback.